products/bash/product.yaml

```diff
@@ -0,0 +1,29 @@
+info:
```

Comment:
Looks like this file was added accidentally

Reply:
yes this files needs to be deleted